lib/bootstrap_form/components/hints.rb

```diff
@@ -38,6 +38,10 @@ def get_help_text_by_i18n_key(name)
         help_text
       end
 
+      def object_class
+        object.class.to_s == "Ransack::Search" ? object.klass : object.class
```

Comment:
Would it make sense to do something like:

```ruby
object.klass || object.class
```

?

This way the _contract_ is: if `object.klass` is defined, it takes precedence over the actual `class`.

Reply:
Thank you for the advice.
I think it is a very good idea. I also considered of the same idea first.

However, the implementation of `object.klass` is free, I felt that there is a possibility that an unintended value would come in and break the code on the library user side. So only in the case of Ransack to limit the impact.